### PR TITLE
added amin function for Paddle mathematical functions

### LINKED ISSUE #22448: amin

### PR DESCRIPTION
**Pull request by Seif-Mohamed1**
Close #22448

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by Seif-Mohamed1** [comment]
amin issue #22448

**Comment by Seif-Mohamed1** [comment]
I look forward to reviewing my pull request

**Comment by Seif-Mohamed1** [comment]
Can you please reviewing my pull request

File: ivy/functional/frontends/paddle/tensor/math.py [hunk 1]
```diff
@@ -492,6 +492,35 @@ def tan(x, name=None):
...
+def stanh(x, scale_a=0.67, scale_b=1.7159, name=None):
+    # TODO this function will be simplified as soon as the ivy.stanh(x,a,b) is added
+    exp_ax = ivy.exp(ivy.multiply(scale_a, x))
+    exp_minus_ax = ivy.exp(ivy.multiply(-scale_a, x))
+    numerator = ivy.subtract(exp_ax, exp_minus_ax)
+    denominator = ivy.add(exp_ax, exp_minus_ax)
+    ret = ivy.multiply(scale_b, ivy.divide(numerator, denominator))
+    return ret
+
+
```
Comment (MahmoudAshraf97) [inline ivy/functional/frontends/paddle/tensor/math.py:505]: remove functions that are not related to your code

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**